securedrop_proxy/entrypoint.py

```diff
@@ -6,18 +6,38 @@
 # called with exactly one argument: the path to its config file. See
 # the README for configuration options.
 
-import sys
 import json
-import uuid
+import logging
+import os
 import subprocess
+import sys
+import uuid
+
+from logging.handlers import TimedRotatingFileHandler
 
-from . import proxy
-from . import config
-from . import callbacks
-from . import main
+from securedrop_proxy import callbacks
+from securedrop_proxy import config
+from securedrop_proxy import main
+from securedrop_proxy import proxy
+from securedrop_proxy.version import version
+
+
+DEFAULT_HOME = os.path.join(os.path.expanduser("~"), ".securedrop_proxy")
 
 
 def start():
+    '''
+    Set up a new proxy object with an error handler, configuration that we read from  argv[1], and
+    the original user request from STDIN.
+    '''
+    try:
+        configure_logging()
+    except Exception as e:
+        print(e)
+        return
+
+    logging.info('Starting SecureDrop Proxy {}'.format(version))
```

Comment:
Perhaps setting to debug here (and other similar messages that don't provide any request-specific information) would make sense?